vignettes/getting-started.Rmd

```diff
@@ -174,7 +174,7 @@ param_df1
 
 Much of the benefit of `rbabylon` is leveraged in the model iteration workflow, and the run log summaries that can be created afterwards. For example, imagine you look at these model results and want to begin iterating on them with a new model. 
 
-Imagine you are now in a new R session and no longer have your `mod1` object. You can easily rebuild it from the YAML file on disk. *NOTE:* You can pass either a character or integer to `read_model()`, `submit_model()`, etc. to identify your model, assuming that you have set your model directory correctly.
+If you are now in a new R session and no longer have your `mod1` object, you can easily rebuild it from the YAML file on disk. *NOTE:* You can pass either a character or integer to `read_model()`, `submit_model()`, etc. to identify your model, assuming that you have set your model directory correctly.
```

Comment:
Right. I almost removed some of this because of that, but it is technically still correct as of 0.8.0.  We'll definitely need an issue in 0.9.0 to clean up all the vignettes to reflect new functionality.